snippets/RPNSolver.md

```diff
@@ -0,0 +1,35 @@
+### RPNSolver
+
+Solves the given reverse polish notation
+```js
+const RPNSolver = RPN => {
+  const isNumeric = str => !isNaN(parseFloat(str)) && isFinite(str);
+  const isOperator = str => ['*','-','+','/','**'].includes(str)
+  let stack = [];
+  let solve  = RPN.replace(/\^/g,'**').split(/\s+/g);
+  solve.forEach(symbol => {
+    if(isNumeric(symbol)) {stack.push(symbol)}
+          else if (isOperator(symbol)){
```

Comment:
You could definitely save a line by inlining the check in this statement.

Reply:
Inlining means?

Reply:
`else if (['*','-','+','/','**'].includes(symbol))`